sys-kernel/linux-headers/Manifest

```diff
@@ -1,2 +1,2 @@
-EBUILD linux-headers-3.10.73.ebuild 1161 BLAKE2B b44eac62853b663f77744efeecbf624c15fa94e71a7e94805174426e9e6fd224100ce784ae5866ef9db9969471c397bb685531c0cc530e87e085218697979b1d SHA512 c2bd268881a30d59741aad0be1d422b341b7ca3d9cc811b7f2d22befb90b13b114b376901fc6f37984176de0bd666afb36cf565ed98cec32c78c713d5a298e19
```

Comment:
I don't think we should replace the 3.10 headers for angler with that for the new device.  Consider adding a new ebuild with better versioning, or splitting the header package altogether.

Reply:
I don't remember why I didn't do this earlier. Sure, will do that.